### PR TITLE
TASK-44: Add cancel button in my events page

### LINKED ISSUE #162: TASK-44: Show 'Cancel' Button for My Events
**Task Description**

As a developer, I want to be able to cancel events that I've created.

**Note**: It is not a functional cancelling of the event. It is just the button styling.

**Acceptance Criteria**

- [ ] Mull button should show "cancel" for events I created
- [ ] Documentation present where necessary
- [ ] Unit and UI tests are implemented for every component and function
- [ ] Must match [Mockup](create-mockup-link)

### PR DESCRIPTION
**Pull request by leojrsilao**
This PR closes #162.

To test this PR:
Go to the my events tab, click on an event you hosted and click on cancel event.

### CODE REVIEW

File: apps/mull-ui/src/app/pages/event-page/event-page-info/event-page-info.scss [hunk 1]
```diff
@@ -9,20 +9,42 @@
...
+  .cancel-event-title {
+    align-self: center;
+  }
+  .close-button {
+    position: absolute;
+    bottom: 22.5rem;
+    left: 13.75rem;
+    cursor: pointer;
+  }
+}
```
Comment (JimLau49) [inline apps/mull-ui/src/app/pages/event-page/event-page-info/event-page-info.scss]:
```suggestion
.cancel-modal-container {
  position: relative;
  display: flex;
  flex-direction: column;
  justify-content: center;
  overflow:hidden;
  .cancel-event-title {
    align-self: center;
  }
  .close-button {
    position: absolute;
    bottom: 21.75rem;
    left: 12.15rem;
    cursor: pointer;
  }
}
```

image

**Review by JimLau49 — CHANGES_REQUESTED** [review]
Nice work but I have some changes,

Firstly, the X button doesn't seem to be aligned for me. This may be because my resolution scaling of my monitor is not 16:9
image

Additionally there also seems to be overflow caused by the X button positioning which is leading to a scrollbar appearing in the modal 

image

I'll leave some suggestions as to how to fix this below

**Review by JimLau49 — APPROVED** [review]
🚀
**Acceptance Criteria**

- [x] Mull button should show "cancel" for events I created
- [x] Documentation present where necessary
- [x] Unit and UI tests are implemented for every component and function
- [x] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2079%3A74708)

**Review by osahtout — APPROVED** [review]
- [X] Does match the mockup that I don't remember making
- It's nice that there's no functionality because I think we need to discuss as a team what happens when cancelling an event

**Pull request merged**